proto/tendermint/types/params.pb.go

```diff
@@ -591,6 +660,30 @@ func (this *HashedParams) Equal(that interface{}) bool {
 	}
 	return true
 }
+func (this *ABCIParams) Equal(that interface{}) bool {
+	if that == nil {
+		return this == nil
+	}
+
+	that1, ok := that.(*ABCIParams)
+	if !ok {
+		that2, ok := that.(ABCIParams)
+		if ok {
+			that1 = &that2
+		} else {
+			return false
+		}
+	}
+	if that1 == nil {
+		return this == nil
+	} else if this == nil {
+		return false
+	}
+	if this.VoteExtensionsEnableHeight != that1.VoteExtensionsEnableHeight {
+		return false
+	}
+	return true
```

Comment:
would 
```suggestion
	return this.VoteExtensionsEnableHeight != that1.VoteExtensionsEnableHeight
```
be idiomatic?

Reply:
Same here, autogenerated code